package.json

```diff
@@ -72,6 +72,7 @@
     "file-loader": "^4.1.0",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
+    "immutable": "^4.0.0-rc.12",
```

Comment:
Is this still needed?

Reply:
I had to add it as a dev dependency since we have slate as a dev dep for the demo/tests and slate needs immutable as a peer dependency.